### DISCUSSION TITLE
New commenting commands `gco`, `gcO` and `gcA` arent working for me

### DISCUSSION THREAD

**Discussion by mangelozzi** [neovim/neovim#29532] [Q&A] [answered]
From `:help commenting`

```
  *gco* - Inserts comment below and enters INSERT mode
  *gcO* - Inserts comment above and enters INSERT mode
  *gcA* - Inserts comment at the end of line and enters INSERT mode
```

But I cant get them to work, even with `nvim --clean`, do they work for any1 else? 

I am using 
```
NVIM v0.11.0-dev-322+ge7020306a
Build type: RelWithDebInfo
LuaJIT 2.1.1716656478
Run "nvim -V1 -v" for more info
```
Which is a few hours old.

**Comment by zeertzjq** [comment, marked as answer]
There are no such commands, and there is no such text at `:help commenting`.

**Comment by mangelozzi** [comment]
Thanks, must have been reading documentation in neovim thinking it was about the built in commenting, but must have been a plug in's docs.
